includes/class-meta-handler.php

```diff
@@ -205,7 +205,7 @@ public static function get_meta( $id, $key, $single_only = true, $callable = nul
 			$callable
 		);
 
-		if ( is_int( $id ) ) {
+		if ( is_int( $id ) || is_string( $id ) ) {
```

Comment:
Don't think this will work. Entire purpose of the check is to see if it's a number.

Let's just use `is_numeric()` instead.

Reply:
Yeah sure that's fine
```suggestion
		if ( is_numeric( $id ) ) {
```